JASP-Engine/JASP/R/reliabilityAnalysisBayesian.R

```diff
@@ -64,19 +64,35 @@ reliabilityBayesian <- function(jaspResults, dataset, options) {
 
 .reliabilityCheckErrors <- function(dataset, options) {
 
-  .hasErrors(dataset = dataset, perform = "run",
+  # check for existing inverse
+  .checkInverse <- function() {
+    if (length(options[["variables"]]) > 2) {
+      use.cases <- "everything"
+      if (any(is.na(dataset))) {
+        if (options[["missingValues"]] == "excludeCasesPairwise") 
+          use.cases <- "pairwise.complete.obs"
+        else if (options[["missingValues"]] == "excludeCasesListwise")
+          use.cases <- "complete.obs"
+      }
+      if (!("matrix" %in% class(try(solve(cov(dataset, use = use.cases)),silent=TRUE)))) {
+        return(gettext("Data covariance matrix is not invertible"))
```

Comment:
rather than checking for "not a matrix" it makes more sense to check for is a try-error imo:
```suggestion
      if (isTryError(try(solve(cov(dataset, use = use.cases)),silent=TRUE))) {
        return(gettext("The covariance matrix of the data is not invertible"))
```

Reply:
Well it seems to me that there is not function "isTryError"